templates/liquid/Measure.liquid

```diff
@@ -49,7 +49,7 @@
                             {% if identifier.type.coding.display.exists() %}
                                 {{identifier.type.coding.display}}{% if identifier.assigner.display.exists() %} ({{identifier.assigner.display}}){% endif %} Identifier:
                             {% else %}
-                                {{identifier.type.coding.code}}{% if identifier.assigner.display.exists() %} ({{identifier.assigner.display}}){% endif %} identifier:
+                                {{ "my great title" | capitalize }}{% if identifier.assigner.display.exists() %} ({{identifier.assigner.display}}){% endif %} identifier:
```

Comment:
Is this a leftover? This doesn't seem right?

Reply:
Yeah, overlooked that. Leftover from trying to capitalize the identifier but forgot to take it out.